packages/manager/src/features/Databases/DatabaseDetail/DatabaseSummary/DatabaseSummaryClusterConfiguration.style.ts

```diff
@@ -0,0 +1,61 @@
+import { styled } from '@mui/material/styles';
+import Grid2 from '@mui/material/Unstable_Grid2/Grid2';
+import { Box } from 'src/components/Box';
+import { Typography } from 'src/components/Typography';
+
+export const StyledGridContainer = styled(Grid2, {
+  label: 'StyledGridContainer',
+})(({ theme }) => ({
+  '&>*:nth-of-type(even)': {
+    boxShadow: `inset 0px -1px 0px 0 ${
+      theme.palette.mode === 'dark'
+        ? theme.color.white
+        : theme.palette.grey[200]
+    }`,
+  },
+  '&>*:nth-of-type(odd)': {
+    boxShadow: `inset 0px -1px 0px 0 ${theme.color.white}`,
+    marginBottom: '1px',
+  },
+  boxShadow: `inset 0 -1px 0 0 ${
+    theme.palette.mode === 'dark' ? theme.color.white : theme.palette.grey[200]
+  }, inset 0 1px 0 0 ${
+    theme.palette.mode === 'dark' ? theme.color.white : theme.palette.grey[200]
+  }, inset -1px 0 0 ${
+    theme.palette.mode === 'dark' ? theme.color.white : theme.palette.grey[200]
+  }`,
+}));
+
+export const StyledLabelTypography = styled(Typography, {
+  label: 'StyledLabelTypography',
+})(({ theme }) => ({
+  background:
+    theme.palette.mode === 'dark'
+      ? theme.bg.tableHeader
+      : theme.palette.grey[200],
+  color: theme.palette.mode === 'dark' ? theme.color.grey6 : 'inherit',
+  fontFamily: theme.font.bold,
+  height: '100%',
+  padding: '4px 15px',
+}));
+
+export const StyledValueBox = styled(Box, {
+  label: 'StyledValueBox',
+})(({ theme }) => ({
+  '& > button': {
+    marginTop: '-5px',
+  },
+  '& > div': {
+    alignSelf: 'flex-start',
+    verticalAlign: 'sub',
+  },
+  '& button.MuiButtonBase-root': {
+    marginTop: '0',
+  },
+  '& button.copy-tooltip': {
+    marginTop: '-3px',
+  },
+  color: theme.palette.mode === 'dark' ? theme.color.grey8 : theme.color.black,
+  display: 'flex',
+  padding: '3px 5px',
+}));
```

Comment:
Could we do something like the following to improve alignment (vertically centered values), make use of flex, and avoid the need for another nested component within the Grid item by replacing the parent of `StyledValueBox` with:

```suggestion
export const StyledGridValue= styled(Grid2, {
  label: 'StyledGridValue',
})(({ theme }) => ({
  alignItems: 'center',
  color: theme.palette.mode === 'dark' ? theme.color.grey8 : theme.color.black,
  display: 'flex',
  padding: `0 ${theme.spacing()}`,
}));
```

I was seeing some weirdness with the icons existing outside the grid container at really small screen sizes (which is also currently happening), so might need to make some adjustments to this still.

![Screenshot 2024-10-16 at 4 07 16 PM](https://github.com/user-attachments/assets/e3e958f4-ee0e-4507-bf33-8aa29533461b)